lib/events/definitions.ts

```diff
@@ -29,6 +29,7 @@ export interface WatchEventCallbacks<
   paired: (paired: boolean) => void;
   reachability: (reachable: boolean) => void;
   'user-info': (payload: P[]) => void;
+  'send-error': (payload: P) => void;
```

Comment:
Same for the JS side - I think it should mention user info specifically e.g. `transfer-user-info-error` or something like that